var/spack/repos/builtin/packages/p4est/package.py

```diff
@@ -42,8 +42,8 @@ class P4est(Package):
     depends_on('mpi')
     depends_on('zlib')
 
-    def install(self, spec, prefix):
-        options = [
+    def configure_args(self):
```

Comment:
Looks like you win, but GitHub isn't complaining about any conflicts? We must have identical changes.